app/views/workflows/_requests.html.erb

```diff
@@ -10,7 +10,7 @@
     </tr>
   </thead>
   <tbody>
-    <% @workflow.data_requests.each do |data_request| %>
+    <% @workflow.data_requests.order(created_at: :desc).each do |data_request| %>
```

Comment:
Ordering can be slow across large sets even if you have an index (especially slow if you don't), is the number of `data_requests` large? Will this slow down over time as more data requests are logged?

Reply:
i can't imagine a situation where there would be more than a hundred or so data requests for a given workflow or project, but i could be wrong.

Reply:
I don't think the order_by can possibly be getting executed in the database because we're using pundit to scope them before they get here.

Reply:
Interesting. Does pundit return an AR scope or a result set?

Reply:
#order is an AR method. If this was being done to an array, I think you'd have to be using Array#sort_by with a block.

Reply:
Good point. Still, I think the db is smart enough to filter before sorting so while we can keep an eye on this, it's unlikely to be a problem any time soon.

Reply:
It might not need pundit to scope the data requests because if you weren't able to view the workflow (by virtue of being an owner/collaborator/admin) you wouldn't even be able to load the view that includes this template.